File: java-cassandra-app/src/test/java/com/sapient/learning/bdd/RunCucumberTest.java
package com.sapient.learning.bdd;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(
		plugin = { "pretty", "json:target/cucumber-reports/integ-test-report.json" }, 
		features = "classpath:features", 
		monochrome = true)
public class RunCucumberTest {
	
}

File: backup/kotlin-cassandra-app/src/test/kotlin/com/sapient/learning/bdd/RunCucumberTest.kt
package com.sapient.learning.bdd

import org.junit.runner.RunWith
import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber

@RunWith(Cucumber::class)
@CucumberOptions(plugin = ["pretty"], features = ["classpath:features"])
class RunCucumberTest {
	
}